decidim-proposals/config/locales/en.yml

```diff
@@ -95,6 +95,8 @@ en:
         proposal:
           fields:
             category: Category
+            comments: Comments
```

Comment:
Should this be "Comments count"?

Reply:
I've followed the same naming as "votes: Votes". If "Comments count" is prefered I can change

Reply:
Ah, I see. Let's leave it as it is then! 😄